salt/minion.py

```diff
@@ -567,8 +567,16 @@ def eval_master(self,
                         break
                     except SaltClientError as exc:
                         last_exc = exc
-                        msg = ('Master {0} could not be reached, trying '
-                               'next master (if any)'.format(opts['master']))
+                        pki_dirname = os.path.dirname(opts['pki_dir'])
+                        if exc.strerror.startswith(
```

Comment:
Is `startswith` the best way to check for this? Or, rather, would a smaller portion of the message make more sense to check for?

Reply:
@rallytime yes, we do need to use startswith because we need to check that the message starts with `Could not access <path>`. But, the logic there was incorrect, I was doing the startswith on the message I wanted to log. D'oh!

I've updated and made the startswith more general so we aren't just looking at the PKI dir.